tools/package.json

```diff
@@ -29,12 +29,14 @@
   "main": "./lib/azure.js",
   "license": "(MIT OR Apache-2.0)",
   "dependencies": {
+    "@azure/ms-rest-azure-js": "^2.1.0",
+    "@azure/ms-rest-nodeauth": "^3.1.1",
```

Comment:
Should we use this? [in the doc](https://www.npmjs.com/package/@azure/ms-rest-nodeauth) its says its no longer maintained, it suggests identity i think for this case, maybe its not possible, but i wanted to ask.

Reply:
yes, you are right for the @azure/ms-rest-azure-js library, but in the case of @azure/ms-rest-nodeauth I'm not sure, because it's being used in other parts of bb

Reply:
I found the @azure/ms-rest-nodeauth issue I'll check azure identity